certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -183,7 +183,7 @@ def main():
         raise ValueError('Invalid server value {0}, should be one of {1}'
                          .format(server_type, possible_values))
 
-    acme_server = ACMEServer(server_type, [], False)
+    acme_server = ACMEServer(server_type, [], http_proxy=False, stdout=True)
     process = None
```

Comment:
Can we remove this unused variable and just change the body of `except KeyboardInterrupt` below to `pass`?

Reply:
Fixed.